modules/nlu/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "nlu",
   "fullName": "Natural Language Understanding",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
I would make this minor change rather than a patch

Reply:
:+1: